beacon-chain/core/state/validity_conditions.go

```diff
@@ -17,6 +17,7 @@ import (
 // IsValidBlock verifies a block is valid according to the ETH 2.0 specification for
 // validity conditions taking into consideration attestation processing and more.
 // TODO(#781): Refactor with the new spec validity conditions.
+// Will be Deprecated
```

Comment:
Please go ahead and mark as deprecated.

// Deprecated: <insert reason>

Reply:
Please mark this as deprecated by prefixing it with `Deprecated:`. This tells go linter that the method _is_ deprecated.